### DISCUSSION TITLE
Does it REALLY make sense to give Shorts its own layouts based on its regular video pages' layouts?

### DISCUSSION THREAD

**Discussion by MesaBlack** [lightbeam24/CustomTube#170] [Q&A]
![image](https://github.com/lightbeam24/CustomTube/assets/132141735/2239e50e-c115-49ee-b1c5-eab8fe873af6)
(Taken on 2008 layout)

**Comment by lightbeam24** [comment]
I don't know what you're talking about. CustomTube doesn't even modify the shorts page (at least not intentionally). And if a CustomTube update were to do anything to the shorts page, it would be removing it altogether (as in redirecting the user to the regular watch page).

**Comment by MesaBlack** [comment]
Like the style of a shorts page is the style of a regular video watch page
from its respective era/

**Comment by lightbeam24** [comment]
> Like the style of a shorts page is the style of a regular video watch page from its respective era/
> […](#)
> On Thu, Feb 1, 2024 at 8:27 PM lightbeam24 ***@***.***> wrote: I don't know what you're talking about. CustomTube doesn't even modify the shorts page (at least not intentionally). And if a CustomTube update were to do anything to the shorts page, it would be removing it altogether (as in redirecting the user to the regular watch page). — Reply to this email directly, view it on GitHub <[#170 (comment)](https://github.com/lightbeam24/CustomTube/discussions/170#discussioncomment-8341381)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A7QFFJ4D3DQG4PPLYLZVAOTYRRFJBAVCNFSM6AAAAABCV5ZFKWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNBRGM4DC> . You are receiving this because you authored the thread.Message ID: ***@***.***>

So basically you want the shorts page to be removed in favor of just having the regular watch page?

**Reply by MesaBlack** [reply]
what no. Like the shorts page had the same design language as the regular watch page

**Reply by lightbeam24** [reply]
Sorry, but this is not planned. I am of the opinion that the Shorts page is horrible and should not exist on desktop YouTube, and I would assume most CustomTube users feel the same way (I haven't spoken to many people who don't hate YouTube Shorts). With that in mind, modifying the shorts page will require far too much effort, as I would have to learn my way around the ever-changing HTML of a page that I never use.

Like I said, if I'm going to do anything to the Shorts page, it's removing it entirely.

**Comment by MesaBlack** [comment]
oh. ok.
